workspaces/tech-insights/.changeset/metal-sloths-wave.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage-community/plugin-tech-insights-backend-module-jsonfc': patch
+'@backstage-community/plugin-tech-insights-backend': patch
+'@backstage-community/plugin-tech-insights-common': patch
+'@backstage-community/plugin-tech-insights-node': patch
+'@backstage-community/plugin-tech-insights': patch
+---
+
+Only show entities with results.
+Additionally, setup dev environment and add some extension examples and update dependencies.
```

Comment:
```suggestion
'@backstage-community/plugin-tech-insights': patch
---

Added a filter to `ScorecardsPage` to only shows entities with results.
```

Then let's create a separate changeset for all other packages saying `Updated dependencies`

Reply:
Good idea, will separate.

Reply:
done